### PR TITLE
fix translations, earlier no translation was loading

### PR DESCRIPTION
**Pull request by umarmughal824**
Fix translations and version bump.

Only following translation are availabe in .po files

- Hide/Show/Print/Download Transcript
and only these translations need to check for this PR.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @umarmughal824! I've created [OSPR-2898](https://openedx.atlassian.net/browse/OSPR-2898) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by umarmughal824** [comment]
@xitij2000 will you please review this PR?

**Comment by giovannicimolin** [comment]
The xblock translations are not working.

- [x] I tested this:
1. Installed the xbock-ooyala on my devstack (with branch `umar/fix-translations`)
2. Added it to advanced modules on the course
3. Created a new unit and added xbock-ooyala in it.
4. Tested with `en` translations (all strings are ok)
5. Tested with `ar` translations (`SELECT TO INMUTE` was in english)
6. Tested with `es-419` (`SELECT TO INMUTE` was in english)
7. Tested with the other languages and the only string shown in the xblock was untranslated (this "Select to unmute button")

- [ ] I've read the code

Screenshot showing the missing translation: https://imgur.com/a/40nnHew

**Comment by xitij2000** [comment]
@umarmughal824 This PR also seems to be removing the bitmovin file. Please keep the scope of this PR to just translations, that is all that is being tested here. The other changes can come in a separate PR. 

Also please update the description of the PR to clearly indicate what this change is supposed to accomplish, since this is just about fixing the translations for "Hide transcript" please indicate that in the PR description.

**Comment by umarmughal824** [comment]
@xitij2000 I have updated the PR description.

**Comment by umarmughal824** [comment]
@giovannicimolin please see the updated description and test it on the given description and this string **SELECT TO UNMUTE** is not a part of **ooyala**.

**Comment by umarmughal824** [comment]
@xitij2000 I have recheck all the files being changes in the PR. I did not found any **bitmovin file** here. It's only dealing with translations. Please review it.

**Comment by giovannicimolin** [comment]
@umarmughal824 Thanks for updating the PR description, I'm not that familiar with this code. 
I'll review it again shortly.

**Comment by xitij2000** [comment]
@umarmughal824 You can see the change here: https://github.com/edx-solutions/xblock-ooyala/pull/93/files#diff-d1f2021e26a240a4f4dcd888dd603f09L156
This should not be part of the PR.

File: ooyala_player/ooyala_player.py [hunk 1]
```diff
@@ -31,19 +31,18 @@
 from .overlay import OoyalaOverlay
 from .tokens import generate_player_token
 from .transcript import Transcript
-from .utils import render_template, _, resource_url
+from .utils import render_template, _, resource_url, I18NService
 
 # Globals ###########################################################
 
 log = logging.getLogger(__name__)
-BIT_MOVIN_PLAYER_PATH = 'public/js/vendor/ooyala/bitmovinplayer.swf'
```
Comment (giovannicimolin) [inline ooyala_player/ooyala_player.py]: Is this related to translations?

File: ooyala_player/ooyala_player.py [hunk 2]
```diff
@@ -153,18 +152,16 @@ def student_view(self, context):
         # Skin file path according to block type
         if hasattr(self, 'lightchild_block_type'):
             json_config_url = resource_url(OoyalaPlayerLightChildBlock.lightchild_block_type, SKIN_FILE_PATH)
-            bit_movin_player_url = resource_url(OoyalaPlayerLightChildBlock.lightchild_block_type, BIT_MOVIN_PLAYER_PATH)
```
Comment (giovannicimolin) [inline ooyala_player/ooyala_player.py]: Are these changes related to translations? If not, revert these lines.

File: ooyala_player/ooyala_player.py [hunk 3]
```diff
@@ -153,18 +152,16 @@ def student_view(self, context):
...
         else:
             json_config_url = resource_url(self.scope_ids.block_type, SKIN_FILE_PATH)
-            bit_movin_player_url = resource_url(self.scope_ids.block_type, BIT_MOVIN_PLAYER_PATH)
```
Comment (giovannicimolin) [inline ooyala_player/ooyala_player.py]: Is this related to translations?

File: ooyala_player/ooyala_player.py [hunk 4]
```diff
@@ -182,12 +179,10 @@ def student_view(self, context):
             'autoplay': self.autoplay,
             'config_url': json_config_url,
             'complete_percentage': COMPLETION_VIDEO_COMPLETE_PERCENTAGE,
-            'bit_movin_player': bit_movin_player_url,
```
Comment (giovannicimolin) [inline ooyala_player/ooyala_player.py]: Same as above

File: ooyala_player/ooyala_player.py [hunk 5]
```diff
@@ -182,12 +179,10 @@ def student_view(self, context):
...
         })
 
         JS_URLS = [
             self.local_resource_url(self, 'public/build/player_all.min.js'),
-            self.local_resource_url(self, 'public/js/vendor/ooyala/bit_wrapper.js'),
```
Comment (giovannicimolin) [inline ooyala_player/ooyala_player.py]: Is this related to translations?

File: ooyala_player/ooyala_player.py [hunk 6]
```diff
@@ -457,6 +453,7 @@ def load_transcript(self, data, suffix=''):
                 api_key=self.get_attribute_or_default('api_key_3play'),
                 threeplay_id=threeplay_id,
                 transcript_id=transcript_id,
+
```
Comment (giovannicimolin) [inline ooyala_player/ooyala_player.py]: Remove this whitespace

File: ooyala_player/transcript.py [hunk 7]
```diff
@@ -186,10 +188,10 @@ def get_transcript_by_threeplay_id(api_key, transcript_id, threeplay_id):
 
         return transcript_content
 
-    def render(self):
+    def render(self, i18n_service=None):
         """
         Render in interactive transcript
         """
-        return render_template('templates/html/ooyala_transcript.html', {
+        return loader.render_django_template('templates/html/ooyala_transcript.html', {
```
Comment (giovannicimolin) [inline ooyala_player/transcript.py]: Can you format this code to follow the edX python guidelines?
```
return loader.render_django_template(
    'templates/html/ooyala_transcript.html', 
    {
        'self': self
    }, 
    i18n_service=i18n_service
)
```

**Comment by umarmughal824** [comment]
@xitij2000 this was mistakenly removed I guess. So I have added it back.

File: ooyala_player/ooyala_player.py (hunk 1 above)
Comment (umarmughal824) [reply ooyala_player/ooyala_player.py]: nope.

File: ooyala_player/ooyala_player.py (hunk 2 above)
Comment (umarmughal824) [reply ooyala_player/ooyala_player.py]: nope

File: ooyala_player/transcript.py (hunk 7 above)
Comment (umarmughal824) [reply ooyala_player/transcript.py]: what guildelines you are talking about? Can you share the link of it?

Comment (giovannicimolin) [reply ooyala_player/transcript.py]: These: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html :grin:

Comment (umarmughal824) [reply ooyala_player/transcript.py]: see what I have here.
https://imgur.com/a/XkZnvlX

**Comment by umarmughal824** [comment]
@giovannicimolin its updated.

**Comment by giovannicimolin** [comment]
@umarmughal824 Can you provide me testing instructions and some content with transcripts? 
I have no way of testing these features as of now.

**Comment by xitij2000** [comment]
@giovannicimolin Ah, I think this will require entering the 3play API key. I have that set up so I'll test out this PR.
